Content.Shared/Chat/Prototypes/EmotePrototype.cs

```diff
@@ -81,5 +81,6 @@ public enum EmoteCategory : byte
     Invalid = 0,
     Vocal = 1 << 0,
     Hands = 1 << 1,
+    GlobalCustom = 1 << 7,
```

Comment:
I felt it would be more appropriate since the General category also have maxvalue. If there is a "Tail" category or something similar later then I would feel weird having it like

```
Vocal = 1 << 0,
Hands = 1 << 1,
GlobalCustom = 1 << 2,
Tail = 1 << 3,
```